lib/vector-core/src/tls/incoming.rs

```diff
@@ -446,6 +447,31 @@ impl From<X509> for CertificateMetadata {
     }
 }
 
+#[derive(Clone)]
```

Comment:
this changes also related to the `cargo check`?

Reply:
Yes